packages/legend-application-query/src/components/Core_LegendQueryApplicationPlugin.tsx

```diff
@@ -595,7 +596,13 @@ export class Core_LegendQueryApplicationPlugin extends LegendQueryApplicationPlu
                 </div>
               );
             }
-            return undefined;
+            return (
+              <div className="query-editor__dataspace__header">
+                <div className="query-editor__header__content__main query-editor__header__content__title__text query-editor__header__content__title__text--unsaved">
+                  Unsaved Query
```

Comment:
Mhm users might mistake this for an actual query. Whats wrong with not showing anything here ?

Reply:
Please check the internal jira ticket raised by John